doc/topics/releases/3005.rst

```diff
@@ -1,7 +1,135 @@
++--------------+---------------------+---------------------+
+| Debian 10    |         yes         |        yes          |
++--------------+---------------------+---------------------+
+| Debian 11    |         yes         |        yes          |
++--------------+---------------------+---------------------+
+| Raspbian 9   |         no          |        yes          |
++--------------+---------------------+---------------------+
+| Raspbian 10  |         no          |        yes          |
++--------------+---------------------+---------------------+
+| Raspbian 11  |         no          |        yes          |
++--------------+---------------------+---------------------+
+| Fedora 35    |         yes         |        yes          |
++--------------+---------------------+---------------------+
+| Fedora 36    |         yes         |        yes          |
++--------------+---------------------+---------------------+
+| MacOS        |         yes         |        yes          |
++--------------+---------------------+---------------------+
+| Windows      |         yes         |        yes          |
++--------------+---------------------+---------------------+
+
+
+Repo paths
+----------
+
++----------+-----------------------------------------------+-----------------------------------------+
+|     OS   | Tiamat Path                                   | Non-Tiamat Path                         |
++==========+===============================================+=========================================+
+| RHEL     | https://repo.saltproject.io/salt/py3/redhat/  | https://repo.saltproject.io/py3/redhat/ |
++----------+-----------------------------------------------+-----------------------------------------+
+| Ubuntu   | https://repo.saltproject.io/salt/py3/ubuntu/  | https://repo.saltproject.io/py3/ubuntu/ |
++----------+-----------------------------------------------+-----------------------------------------+
+| Debian   | https://repo.saltproject.io/salt/py3/debian/  | https://repo.saltproject.io/py3/debian/ |
++----------+-----------------------------------------------+-----------------------------------------+
+| Raspbian | https://repo.saltproject.io/salt/py3/debian/  | https://repo.saltproject.io/py3/debian/ |
++----------+-----------------------------------------------+-----------------------------------------+
+| Fedora   | Hosted on Fedora Repos                        | Hosted on Fedora Repos                  |
++----------+-----------------------------------------------+-----------------------------------------+
+| MacOS    | https://repo.saltproject.io/salt/py3/osx/     | https://repo.saltproject.io/osx/        |
++----------+-----------------------------------------------+-----------------------------------------+
+| Windows  | https://repo.saltproject.io/salt/py3/windows/ |  https://repo.saltproject.io/windows/   |
++----------+-----------------------------------------------+-----------------------------------------+
```

Comment:
**Question:** You mentioned that we're going to support latest, major, minor for Tiamat. Do we need to list separate URLs for those?

Reply:
Just looked at current for Redhat and it has 'pin to latest, 'pin to major' and 'pin to minor' , so probably not, since we are already stating that currently.